memory/stutter.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author:Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+
+
+import os
+
+from avocado import Test
+from avocado import main
+from avocado.utils import process, archive, memory, build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Stutter(Test):
+
+    """
+    stutter benchmark
+    """
+
+    def setUp(self):
+        '''
+        Build Stutter Test
+        Source:
+        https://github.com/gaowanlong/stutter/archive/master.zip
+        '''
+
+        # Check for basic utilities
+
+        smm = SoftwareManager()
+        if not smm.check_installed("gcc") and not smm.install("gcc"):
+            self.error('Gcc is needed for the test to be run')
+
+        locations = ["https://github.com/gaowanlong/stutter/archive/"
+                     "master.zip"]
+
+        tarball = self.fetch_asset("stutter.zip", locations=locations,
+                                   expire='7d')
+
+        archive.extract(tarball, self.srcdir)
+
+        self.srcdir = os.path.join(self.srcdir, 'stutter-master')
+
+        mem_byte = str(memory.memtotal())
+
```

Comment:
i used it because i think through this code look clean , will remove white spaces where i feel logic should in a block